server/crawler/index.ts

```diff
@@ -188,6 +189,13 @@ function formatBody({
   return body
 }
 
+async function getBrowserPage():Promise<any> {
+  const browser = await puppeteer.launch({
+    headless: false,
```

Comment:
Can we use `headless: true` with custom viewport, user agent and langauge setting?

Reply:
`    await page.setUserAgent(`Mozilla/5.0 (LikerBot, ${LIKECOIN_CURRENT_CODENAME}) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/68.0.3440.106 Safari/537.36`);
          await page.setExtraHTTPHeaders({
            'Accept-Language': 'zh-HK,zh-TW;q=0.9,zh;q=0.8,en-US;q=0.7,en;q=0.6', // bypass Cloudflare
          });` still fail